nixos/modules/programs/zsh/zsh-autosuggestions.nix

```
@@ -56,6 +63,7 @@ in
 
       export ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE="${cfg.highlightStyle}"
       export ZSH_AUTOSUGGEST_STRATEGY=("${cfg.strategy}")
+      ${if !cfg.async then "unset ZSH_AUTOSUGGEST_USE_ASYNC" else ""}
```

```suggestion
      ${lib.optionalString !cfg.async "unset ZSH_AUTOSUGGEST_USE_ASYNC"}
```
we have a convivence function for this

---

yeah, I tried doing that, but then realized it has to be
```
${lib.optionalString (!cfg.async) "unset ZSH_AUTOSUGGEST_USE_ASYNC"}
```
and wasn't sure it was any better than just using `if`.

---

@SuperSandro2000 fixed, thanks.